include/highfive/bits/H5Path_traits_misc.hpp

```diff
@@ -0,0 +1,29 @@
+#include <H5Ipublic.h>
+
+#include "H5Utils.hpp"
+#include "H5Path_traits.hpp"
+
+namespace HighFive{
+
+template <typename Derivate>
+inline PathTraits<Derivate>::PathTraits() {
+    const hid_t hid = static_cast<Derivate*>(this)->getId();
+    hid_t file_id = H5Iget_file_id(hid);
+    if (file_id > 0 and file_id != hid) {
```

Comment:
```suggestion
    if ((file_id > 0) && (file_id != hid)) {
```
https://github.com/BlueBrain/HighFive/pull/417/checks?check_run_id=2085551915#step:5:98
maybe this will appease visual studio compiler...

Reply:
😆 too much python :p